backend/api/Controllers/ParcelController.cs

```diff
@@ -115,8 +123,13 @@ public IActionResult GetParcel(int id)
         [SwaggerOperation(Tags = new[] { "parcel" })]
         public IActionResult AddParcel([FromBody] Model.ParcelModel model)
         {
+            model.Address.Postal = model.Address.Postal.Replace(" ", "");
+            foreach(var item in model.Buildings)
+            {
+                item.Address.Postal = item.Address.Postal.Replace(" ", "");
+            }
             var entity = _mapper.Map<Entity.Parcel>(model);
-
+            
             _pimsService.Parcel.Add(entity);
```

Comment:
Move formatting of postal to service layer instead so that it's handled with any future endpoints.